import-export-cli/integration/apim/apim.go

```diff
@@ -2295,3 +2297,178 @@ func getTokenURL(host string, offset int) string {
 	port := 9443 + offset
 	return "https://" + host + ":" + strconv.Itoa(port) + "/oauth2/token"
 }
+
+// GetAPIPolicies : Get API Policies list of all types from APIM
+func (instance *Client) GetAPIPolicies(t *testing.T, offset, limit string) *APIPoliciesList {
+	var policyListResponse *APIPoliciesList
+
+	getPoliciesURL := instance.publisherRestURL + "/operation-policies"
```

Comment:
Can't we get `/operation-policies` string through a constant? I see that there are 5 occurrences for this. @wasuradananjith WDYT?

Reply:
+1 to use a constant. We can refactor other functions as well (which are used for APIs, API Products and Applications) to have this convention in the future.
However, we might need to introduce a new constant file, because apim.go cannot use testconstants.go file AFAIR because it will introduce a recursive go dependency.. Or else, we can just define the constants on top of apim.go because anyway these constants will only be used inside this file. WDYT @ashera96 ?

Reply:
+1 @wasuradananjith. We can define within apim.go itself since they are only used in this file. Shall we do that improvement then @BLasan? Also, let's refactor other functions to follow the same convention.

Reply:
+1 @ashera96

Reply:
Shall we add a constant `instance.operationPolicyRestURL` to retrieve a URL including the resource path? WDYT ? @wasuradananjith @ashera96

Reply:
Introducing a const `operationPolicyResourcePath` inside the file would solve the problem